packages/account/src/Sections/Security/Passwords/passwords.tsx

```diff
@@ -38,28 +41,13 @@ const Passwords = observer(() => {
 
     return (
         <div className='account__passwords'>
-            <DerivEmail
-                email={email}
-                social_identity_provider={social_identity_provider}
-                is_social_signup={is_social_signup}
-            />
-            <DerivPassword
-                email={email}
-                is_dark_mode_on={is_dark_mode_on}
-                is_social_signup={is_social_signup}
-                is_eu_user={is_eu_user}
-                financial_restricted_countries={financial_restricted_countries}
-                social_identity_provider={social_identity_provider}
-            />
+            <DerivEmail />
+            <DerivPassword />
             {!is_from_derivgo && (mt5_login_list?.length > 0 || !is_mt5_password_not_set) && (
```

Comment:
This condition check (mt5_login_list?.length > 0 || !is_mt5_password_not_set) is repeated. can we assign to a variable and reuse it?